src/containers/NavBar.tsx

```diff
@@ -22,7 +24,41 @@ const Root = styled.div<{ theme: Theme; position: "relative" | "fixed" }>`
   & > button {
     padding: 0 1em;
   }
+  .nav-button {
+    display: none !important;
+    position: absolute;
+    right: 2em;
+    top: 2em;
+    transform: translate(0, -50%);
+  }
+
+  .title-children-container-mobile {
+    display: none;
+  }
+
+  @media (max-width: 768px) {
+    &.nav-open {
+      height: auto;
+      ${({ position }) => position !== "fixed" && `z-index: 1;`}
+    }
+
+    .nav-button {
+      display: block !important;
+    }
+
+    .title-children-container {
+      div {
```

Comment:
This nesting here appears to be causing issues on my machine

Reply:
Whats the reason for nesting the div selector?

Reply:
This is just a css namespace it should be converted to something like: 
`.title-children-container div{
...
}`

So basically every <div> inside the .title-children-container. 
However I changed this line to select every element except the first in mobile resolutions (at the moment resolutions below 768px), and this hide them.